templates/attr.erb

```diff
@@ -1,5 +1,6 @@
 attr_filter <%= @prefix %>.<%= @name %> {
 	key = "%{<%= @key %>}"
 	filename = ${modconfdir}/${.:name}/<%= @name %>
+	relaxed = <%= @relaxed %>
```

Comment:
If this is undef (i.e. it is specified as optional, so that is .. sort of .. possible even though the current default is 'no', this should check if it is set, and only write it to the file if so.

Reply:
That's been updated as well in my repo.